.dir-locals.el

```diff
@@ -0,0 +1,2 @@
+((nil . ((indent-tabs-mode . nil)
```

Comment:
what is this file for?

Reply:
Sets Emacs variables, in this case, to not use tabs